### DISCUSSION TITLE
Livewire V3 - duplicated Alpine instance

### DISCUSSION THREAD

**Discussion by placiu** [alpinejs/alpine#3687] [7. Bugs (archived)]
Hi!,

So, I'm trying the new Livewire V3 which ships with its own Alpine package, I believe. I've also installed Laravel Breeze, which setup Alpine as well (in resources/js/app.js). I came across this issue when using `x-teleport` it renders twice the same content. I know it has something to do with this duplicated packages because if I remove `app.js` initialization it will stop duplicate teleported content, but it breaks other functionalities. 

Has anyone had a similar problem, or am I just doing something wrong?

**Comment by joshhanley** [comment]
@placiu Please open this on the Livewire repo 🙂
